core/src/main/java/org/seedstack/seed/core/internal/configuration/ConfigurationPriority.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2013-2016, The SeedStack authors <http://seedstack.org>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.seedstack.seed.core.internal.configuration;
+
+public class ConfigurationPriority {
+    /**
+     * Environment variables have the highest priority to avoid any override.
+     */
+    public static final int SYSTEM_PROPERTIES = Integer.MAX_VALUE;
+
+    /**
+     * Environment variables have the highest priority to avoid any override.
+     */
+    public static final int ENVIRONMENT_VARIABLES = Integer.MAX_VALUE;
```

Comment:
@adrienlauer 
SYSTEM_PROPERTIES and ENVIRONMENT_VARIABLES have the same priority isn't it a source of potential confusion if the same key is present via both sources ?